src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -299,6 +299,11 @@ module sonic-device_metadata {
                     type string;
                     description "Set source of anchor route";
                 }
+
+                leaf-list orch_zmq_tables {
```

Comment:
>orch_zmq_tables [](http://example.com/codeflow?start=26&length=15)

I understand orch_zmq_tables config is super flexible and could dynamically change orchagent behavior. Do we really need such flexibility? If not, a static config file on disk may be good enough.

Reply:
We need at least 2 feature flags:
Enable/disable improve Route performance with ZMQ.
Enable/disable improve Dash performance with ZMQ.

Currently there is no feature flag for enable/disable Dash ZMQ, it's controlled by the 'subtype' flag, if subtype is 'SmartSwitch' then orchagent will start with ZMQ enabled and Dash tables will use ZMQ.

However, after add Route ZMQ support, orchagent can't use ZMQ flag to identify Dash ZMQ feature status, so we need a Dash ZMQ feature flag.